index.html

```diff
@@ -495,6 +495,462 @@ <h2>
         </p>
       </section>
     </section>
+    <section>
+      <h3>
+        `file_handlers` member
+      </h3>
+      <p>
+        The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+        "manifest">file_handlers</dfn></code> member is a [=list=] that
+        provides instructions for how the app handles file-opening actions that
+        originate outside of the app itself.
+      </p>
+      <p>
+        The management, presentation, and selection of registered file-handling
+        applications is at the discretion of the operating system.
+      </p>
+      <p>
+        To <dfn class="lint-ignore">process the `file_handlers` member</dfn>,
+        given [=ordered map=] |json:ordered map|, [=ordered map=]
+        |manifest:ordered map|, [=URL=] |manifest_url:URL|:
+      </p>
+      <ol class="algorithm">
+        <li>Let |processedFileHandlers:list| be a new [=list=].
+        </li>
+        <li>Set |manifest|["file_handlers"] to |processedFileHandlers|.
+        </li>
+        <li>If |json|["file_handlers"] doesn't [=map/exist=] or
+        |json|["file_handlers"] is not a [=list=], return.
+        </li>
+        <li>[=list/For each=] |entry:ordered map| of |json|["file_handlers"]:
+          <ol>
+            <li>Let |file_handler:ordered map| be [=process a file handler
+            item=] with |entry|, [=manifest/start_url=], [=manifest/scope=],
+            and |manifest_url|.
+            </li>
+            <li>If |file_handler| is failure, continue.
+            </li>
+            <li>[=list/Append=] |file_handler| to |processedFileHandlers|.
+            </li>
+          </ol>
+        </li>
+      </ol>
+      <p>
+        On [=installation=], a user agent SHOULD run the process to [=register
+        file handlers=].
+      </p>
+      <section data-cite="file-system-access">
+        <h2>
+          File Handler Items
+        </h2>
+        <p>
+          Each <dfn data-local-lt="file handler">file handler</dfn> represents
+          a URL in the scope of the application that can handle launches with
+          [=file types=] it accepts. It has the following members:
+        </p>
+        <ul>
+          <li>[=file handler/action=]
+          </li>
+          <li>[=file handler/name=]
+          </li>
+          <li>[=file handler/accept=]
+          </li>
+          <li>[=file handler/icons=]
+          </li>
+        </ul>
+        <p>
+          A user agent can use these members to associate the web application
+          with [=file type=] on the operating system.
+        </p>
+        <p>
+          A <dfn>file type</dfn> can be defined by a [=mime type=] and/or
+          [=file extension=]. A file belongs to a file type if the OS
+          determines it to have a mime type and/or its name ends with a certain
```

Comment:
```suggestion
          determines it to have a MIME type and/or its name ends with a certain
```

I think it's commonly uppercase in prose.

Reply:
[=MIME type=] then, no?

Reply:
done.